src/tike/operators/convolution.py

```diff
@@ -6,11 +6,44 @@
 
 
 class Convolution(Operator):
-    """2D Convolution operator with linear interpolation."""
+    """A 2D Convolution operator with linear interpolation.
+
+    Compute the product two arrays at specific relative positions.
+
+    Attributes
+    ----------
+    nscan : int
+        The number of scan positions at each angular view.
+    fly : int
+        The number of consecutive scan positions that describe a fly scan.
+    nmode : int
```

Comment:
Does it make sense to combine fly and nmode attributes? Both refer to the number of probes that is associated to data collected at a single exposure.

Reply:
No. modes are different illuminations, same scan position hitting the detector together. fly(s?) are the same illuminations different scan position hitting the detector together.